tests/ft_locales/ft_locales.cpp

```diff
@@ -4393,8 +4393,9 @@ void Ft_Locales::checkAvailableLocales()
     requiredLocaleNames << "km";          // "Khmer"
     requiredLocaleNames << "km_KH";       // "Khmer (Cambodia)"
     requiredLocaleNames << "ms";          // "Malay"
-    requiredLocaleNames << "ms_BN";       // "Malay (Brunei)"
-    requiredLocaleNames << "ms_MY";       // "Malay (Malaysia)"
+    // Current device locales doesnt seem to contain these
```

Comment:
What device? What locales?

Quickly googling found this
http://bugs.icu-project.org/trac/changeset/33085
so seems they got deprecated. In such case doesn't make sense to continue requiring them here, even commented out.